.dialyzer.ignore-warnings

```diff
@@ -86,6 +86,8 @@ tate_root':=_, _=>_}, _=>_}
 # using erlang 21 while we are using erlang 20. But we cannot yet update to
 # erlang 21 because we need rox to update
 apps/blockchain/lib/blockchain/block.ex:739: Function create_receipt/6 will never be called
+apps/blockchain/lib/blockchain/account/repo.ex:47: Function commit/1 has no local return
+apps/blockchain/lib/blockchain/account/repo.ex:50: The call 'Elixir.Blockchain.Account.Repo':new(#{<<_:160>>=>#{'balance':=integer(), 'code':=binary(), 'nonce':=integer(), 'storage':=#{'__struct__':='Elixir.MerklePatriciaTree.Trie', 'db':={atom(),_}, 'root_hash':=binary()}}}) will never return since it differs in the 1st argument from the success typing arguments: (#{'__struct__':='Elixir.MerklePatriciaTree.Trie', 'db':={atom(),_}, 'root_hash':=binary()})
```

Comment:
These are the last two warnings that I did not tackle because they seem to be going deeper into other modules. I will see if they have a quick fix, but since the issue wasn't apparent to me, I decided to leave them here for now